src/Runner.Worker/JobExtension.cs

```diff
@@ -137,6 +137,7 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         var portInfo = url.IsDefaultPort ? string.Empty : $":{url.Port.ToString(CultureInfo.InvariantCulture)}";
                         context.SetGitHubContext("url", $"{url.Scheme}://{url.Host}{portInfo}");
                         context.SetGitHubContext("api_url", $"{url.Scheme}://{url.Host}{portInfo}/api/v3");
+                        context.SetGitHubContext("graphql_url", $"{url.Scheme}://{url.Host}{portInfo}/api/graphql");
```

Comment:
this is safe to add, it just for testing against GHES until the URL changes are in all the different repos.

Reply:
going ahead and removing from this PR since it's not related